### DISCUSSION TITLE
Hitting "During Config validation provided topology is either non existing or does not end with .yaml occurred"

### DISCUSSION THREAD

**Discussion by lpoiana** [Templum/rabbitmq-connector#181] [Q&A] [answered]
Hello, 

I'm getting a CrashLoopBackOff of rabbitmq-connector pod due to : "During Config validation provided topology is either non existing or does not end with .yaml occurred."

connector-cfg:
apiVersion: v1
kind: ConfigMap
metadata:
  name: rabbitmq-connector-configmap
  namespace: openfaas
data:
  OPEN_FAAS_GW_URL: "http://xxxxxxxxxx:8080"
  RMQ_TOPICS: "leo,foo,bar"
  RMQ_HOST: "xxxxxx"
  RMQ_PORT: "15672"
  RMQ_USER: "guest"
  RMQ_PASS: "guest"
  PATH_TO_TOPOLOGY: "/home/petre_leonard_poiana/gke/rabbitmqconnector/tpg.yaml"
  REQ_TIMEOUT: "30s"
  TOPIC_MAP_REFRESH_TIME: "30s"

tpg.yaml:
- name: LEO
  topics: [leo, foo, bar]
  declare: true
  type: "direct"
  durable: false
  auto-deleted: false

connector-dep:

.....
        - name: PATH_TO_TOPOLOGY
          valueFrom:
            configMapKeyRef:
              name: rabbitmq-connector-configmap
              key: PATH_TO_TOPOLOGY
.......

Can you help me to bypass this error?

Thanks in advance.

**Comment by Templum** [comment, marked as answer]
Generally speaking, this issue is quite simple, either you don't provide any correct **absolute path** (which is not the case). Or it seems to be more likely the path provided is not available in the running instance.

So in your situation, I would attempt to verify that the folder you try to access does really exist in the running instance. Because from what you shared it is not visible to me how you pass in the topology.yaml, something that was discussed in great length [here](https://github.com/Templum/rabbitmq-connector/issues/94).

**Reply by lpoiana** [reply]
thanks a lot for your help, completely forgot to pass the topology.yaml, my issue is solved
